CHANGELOG.md

```diff
@@ -6,454 +6,473 @@ All notable changes to this project will be documented in this file.
 
 ## Added
 
-- Automate
-  - Import Rake task `OVERWRITE` argument: Setting  `OVERWRITE=true` removes the target domain prior to import
-  - New `extend_retires_on` method: Used by Automate methods to set a retirement date to specified number of days from today, or from a future date.
-  - Service model updates
-    - MiqAeServiceHardware
-    - MiqAeServicePartition
-    - MiqAeServiceVolume
+### Automate
+
+- New methods
+  - `extend_retires_on` method: Used by Automate methods to set a retirement date to specified number of days from today, or from a future date.
+  - `taggable?` to programmatically determine if a Service Model class or instance is taggable.
+- Service model updates
+  - MiqAeServiceHardware
+  - MiqAeServicePartition
+  - MiqAeServiceVolume
+- Generic Objects: Model updates
+  - Associations
+  - Tagging
+  - Service Methods: `add_to_service / remove_from_service`
+  - Service models created for GenericObject and GenericObjectDefinition
```

Comment:
These are new service models and I like having them listed here.  Should they also be listed in the "New Service Models" section?